plugin/pkg/scheduler/factory/plugins_test.go

```diff
@@ -39,3 +39,22 @@ func TestAlgorithmNameValidation(t *testing.T) {
 		}
 	}
 }
+
+func TestPriorityOverFlow(t *testing.T) {
```

Comment:
definetly should have a test confirming it doesnt always overflow.  that would be pretty bad :)